CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
-# v0.12.0 (Unreleased)
+# v0.13.0 (Unreleased)
```

Comment:
conflict in this file

Reply:
it should be v13 unreleased?  you released v12...

Reply:
Yeah it should be v0.13.0

Reply:
There is still a merge conflict in this file